doc/services/zbus/index.rst

```diff
@@ -1,58 +1,74 @@
 .. _zbus:
 
-Zephyr message bus (zbus)
-#########################
+Zephyr Bus (ZBus)
+#################
 
-The :dfn:`Zephyr message bus - Zbus` is a lightweight and flexible message bus enabling a simple way for threads to talk to one another.
+The :dfn:`Zephyr Bus - ZBus` is a lightweight and flexible software bus enabling a simple way for threads to talk to one another.
 
 .. contents::
     :local:
     :depth: 2
 
 Concepts
 ********
-
-Threads can broadcast messages to all interested observers using zbus. Many-to-many communication is possible. The bus implements message-passing and publish/subscribe communication paradigms that enable threads to communicate synchronously or asynchronously through shared memory. The communication through zbus is channel-based, where threads publish and read to and from using messages. Additionally, threads can observe channels and receive notifications from the bus when the channels are modified. The figure below shows an example of a typical application using zbus in which the application logic (hardware independent) talks to other threads via message bus. Note that the threads are decoupled from each other because they only use zbus' channels and do not need to know each other to talk.
+Threads can broadcast messages to all interested observers using ZBus. Many-to-many communication is possible. The bus implements message-passing and publish/subscribe communication paradigms that enable threads to communicate synchronously or asynchronously through shared memory. The communication through ZBus is channel-based, where threads publish and read to and from using messages. Additionally, threads can observe channels and receive notifications from the bus when the channels are modified. The figure below shows an example of a typical application using ZBus in which the application logic (hardware independent) talks to other threads via software bus. Note that the threads are decoupled from each other because they only use ZBus' channels and do not need to know each other to talk.
```

Comment:
I would also consider making this several paragraphs, as it is a bit intimidating to have the introduction to ZBus/zbus be such a big block of text

<img width="660" alt="image" src="https://github.com/zephyrproject-rtos/zephyr/assets/128251/b65bcf3a-68d8-4936-96e3-ec3211083a00">